executor/runcexecutor/executor_linux.go

```diff
@@ -33,22 +34,34 @@ func (w *runcExecutor) run(ctx context.Context, id, bundle string, process execu
 }
 
 func (w *runcExecutor) exec(ctx context.Context, id, bundle string, specsProcess *specs.Process, process executor.ProcessInfo, started func()) error {
-	return w.callWithIO(ctx, id, bundle, process, started, func(ctx context.Context, started chan<- int, io runc.IO) error {
+	isExec := true
```

Comment:
This is somewhat weird now as we are passing a callback that does exec but then we are also passing a boolean to call the related behavior. Should the callback return the killer implementation as well?

Reply:
Yeah, you are correct, I think I can pass in the correct killer now, will update.

Reply:
updated